azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -18,16 +18,16 @@
 class AbstractIoTHubClient(object):
     """A superclass representing a generic client. This class needs to be extended for specific clients."""
 
-    def __init__(self, pipeline):
```

Comment:
As per discussion, we should consider if we want this to also apply the patch to a twin object